packages/binding-modbus/README.md

```diff
@@ -164,7 +164,7 @@ Reads the 8th input register of the unit 1
     "contentType": "application/octet-stream;length=2",
```

Comment:
In modbus package we are making no actual usage of `contentType;length=` parameter so I'm just reverting the example from `quantity` to `length` and it should be good